api/database/data/qa/04_pip_archer_logs.sql

```diff
@@ -0,0 +1,187 @@
+    (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT volunteer_activity_id FROM volunteer_activity WHERE volunteer_activity_name='Helping with raising funds (shop, events…)'),
+    INTERVAL '00:05:00' DAY TO SECOND,
+    '2019-04-14T10:00:00.000'),
+( (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Archers'),
+    (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT volunteer_activity_id FROM volunteer_activity WHERE volunteer_activity_name='Office support'),
+    INTERVAL '00:05:00' DAY TO SECOND,
+    '2019-04-15T10:00:00.000'),
+( (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Archers'),
+    (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT volunteer_activity_id FROM volunteer_activity WHERE volunteer_activity_name='Other'),
+    INTERVAL '00:05:00' DAY TO SECOND,
+    '2019-04-16T10:00:00.000'),
+( (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Archers'),
+    (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT volunteer_activity_id FROM volunteer_activity WHERE volunteer_activity_name='Outdoor and practical work'),
+    INTERVAL '00:05:00' DAY TO SECOND,
+    '2019-04-17T10:00:00.000'),
+( (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Archers'),
+    (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT volunteer_activity_id FROM volunteer_activity WHERE volunteer_activity_name='Professional pro bono work (Legal, IT, Research)'),
+    INTERVAL '00:05:00' DAY TO SECOND,
+    '2019-04-18T10:00:00.000'),
+( (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Archers'),
+    (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT volunteer_activity_id FROM volunteer_activity WHERE volunteer_activity_name='Shop/Sales'),
+    INTERVAL '00:05:00' DAY TO SECOND,
+    '2019-04-19T10:00:00.000'),
+( (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT organisation_id FROM organisation WHERE organisation_name='Twine Testing Archers'),
+    (SELECT user_account_id FROM user_account WHERE email='b@archers.com'),
+    (SELECT volunteer_activity_id FROM volunteer_activity WHERE volunteer_activity_name='Support and Care for vulnerable community members'),
+    INTERVAL '00:05:00' DAY TO SECOND,
+    '2019-04-20T10:00:00.000'),
+--24h------------------------------------------------
```

Comment:
It would be good to get clearer comments on the types of logs that are added here.